docs/infrastructure/deployment-targets/reflected-targets/index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Reflected Targets
+description: Octopus resources created in cloud providers can be discovered automatically by Octopus
+position: 100
+---
+
+Octopus can automatically discover deployment targets during deployment using tags added to cloud resources. These targets that are automatically discovered will also be removed from Octopus when the associated cloud resource is removed. Automatic discovery will take place before the relevant deployment step and can be used as part of deployment processes that create your cloud infrastructure before deployment.
```

Comment:
I feel we can drop the word `automatically` in these sentences - it does not seem to add more clarity / context.

Reply:
> relevant deployment step

This may need a small amount of clarification. Perhaps it could just be simplified to "before deployment steps"

Reply:
```suggestion
Octopus can discover deployment targets during deployments using tags added to your cloud resources. Octopus will remove any discovered targets when the associated cloud resource is removed. Target discovery takes place before deployment steps, and is useful when your deployment process creates your target cloud infrastructure before deploying software to it.
```

Reply:
Serving suggestion